data/bash-completion/s4cmd

```diff
@@ -0,0 +1,70 @@
+_s4cmd ()
+{
+    local IFS=$' \n'
+    local cur prev words cword possibleparams
+    command=${COMP_WORDS[1]}
+    prev=${COMP_WORDS[COMP_CWORD-1]}
+    prevprev=${COMP_WORDS[COMP_CWORD-2]}
+    _init_completion || return
+
+    #echo -e "\nCurrent: $cur : Prev: $prev : Words: $words : Cword: $cword : Command: $command\n"
+    if [[ $cword -eq 1 ]]; then
+        #command
+        COMPREPLY=( $( compgen -W 'ls put get cp mv sync del du' -- "$cur" ) )
```

Comment:
Not sure what is your convention here, but can you make sure the spaces after ( and before ) is consistent?
For example, next line doesn't have any spaces.

Same for other places.